### PR TITLE
WPF-61988 Some short cut key commands are missing in UG

### PR DESCRIPTION
**Pull request by deepathiruppathy01**
## Description:

Added missed short cut key commands for SfDiagram control.

### CODE REVIEW

File: wpf/Diagram/Interaction/Keyboard.md [hunk 1]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
 | Ctrl + D | Duplicate | Copies the selected element from the diagram and pastes the copied element into the diagram. |
 | Delete | Delete | Delete the selected elements in the diagram. |
 | LeftArrow | MoveLeft | MoveLeft (move the selected elements towards left by one pixel). |
+| Shift + LeftArrow | MoveLeft | MoveLeft (move the selected elements towards left by ten pixels). |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:28]: | Shift + LeftArrow | MoveLeft | MoveLeft (move the selected elements towards the left by ten pixels). |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 2]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
 | RightArrow | MoveRight | MoveRight (move the selected elements towards right by one pixel). |
+| Shift + RightArrow | MoveRight | MoveRight (move the selected elements towards right by ten pixels). |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:30]: | Shift + RightArrow | MoveRight | MoveRight (move the selected elements towards the right by ten pixels). |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 3]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
 | UpArrow | MoveUp | MoveUp (move the selected elements towards up by one pixel). |
+| Shift + UpArrow | MoveUp | MoveUp (move the selected elements towards up by ten pixels). |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:32]: | Shift + UpArrow | MoveUp | MoveUp (move the selected elements upwards by ten pixels). |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 4]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
 | DownArrow | MoveDown | MoveDown (move the selected elements towards up by one pixel). |
+| Shift + DownArrow | MoveDown | MoveDown (move the selected elements towards up by ten pixels). |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:34]: | Shift + DownArrow | MoveDown | MoveDown (move the selected elements upwards by ten pixels). |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 5]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| MouseScroll | Zoom | Scrolls the diagram page up and down in vertical direction. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:35]: | MouseScroll | Zoom | Scrolls the diagram page up and down in a vertical direction. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 6]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| Shift + MouseScroll | Zoom | Scrolls the diagram page left and right in horizontal direction. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:36]: | Shift + MouseScroll | Zoom | Scrolls the diagram page left and right in a horizontal direction. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 7]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
 | Ctrl + ] | BringForward | Moves the selected element over the nearest overlapping element. |
-| Ctrl + |Zoom | Zoom in the diagram. |
-| Ctrl - | Zoom | Zoom out the diagram. |
+| Ctrl + PlusKey(+)|Zoom | Zoom in the diagram. |
+| Ctrl - MinusKey(-)| Zoom | Zoom out the diagram. |
 | Ctrl + 0 (number 0) | Reset | To reset horizontal Offset, vertical Offset, and zoom level of the Diagram. |
 | Ctrl + Shift + W | FitToPage | Fit the diagram content into the view with respect to either width, height, or at the whole. |
 | F2 | EditAnnotation | Enable annotation editing mode of the first selected diagram element. |
+| Enter | SelectFocusedItem | Selects the focused item. |
+| Tab | FocusToNextItem | Focus the item which is next to selected item. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:50]: | Tab | FocusToNextItem | Focus on the item next to the selected item. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 8]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| Shift + Tab | FocusToPreviousItem | Focus the item which is previous to selected item. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:51]: | Shift + Tab | FocusToPreviousItem | Focus on the item previous to the selected item. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 9]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| Ctrl + R | RotateRight | Rotates the selected node in clockwise to 90 degrees. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:52]: | Ctrl + R | RotateRight | Rotates the selected node clockwise to 90 degrees. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 10]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| Ctrl + L | RotateLeft | Rotates the selected node in anticlockwise to 90 degrees. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:53]: | Ctrl + L | RotateLeft | Rotates the selected node anticlockwise to 90 degrees. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 11]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| Ctrl + H | HorizontalFlip | Flips the selected node in horizontal direction. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:54]: | Ctrl + H | HorizontalFlip | Flips the selected node in a horizontal direction. |

File: wpf/Diagram/Interaction/Keyboard.md [hunk 12]
```diff
@@ -25,21 +25,37 @@ The following table illustrates list of Commands with key Gesture.
...
+| Ctrl + J | VerticalFlip | Flips the selected node in vertical direction. |
```
Comment (Chrispineagunja) [inline wpf/Diagram/Interaction/Keyboard.md:55]: | Ctrl + J | VerticalFlip | Flips the selected node in a vertical direction. |

**Review by DeepakRajSundar — APPROVED** [review]

**Pull request merged**